services/QuillLMS/app/controllers/teachers/progress_reports/activity_sessions_controller.rb

```diff
@@ -151,15 +152,32 @@ def index
     end
   end
 
+  private def timespent_string(seconds)
```

Comment:
I'd recommend using this Rails ActionView [helper](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words) since it probably covers some weird edge cases.

Reply:
oh cool I didn't know that existed! however, there was a specific format requested for this feature: https://www.notion.so/quill/Add-time-spent-to-reports-e06905d6d9ab477f97c5223c317d9e23#c178ad1090234b01a202fad5cf6cf235

Reply:
That makes sense.  Maybe next time.